frontend/public/components/secrets/create-secret.tsx

```diff
@@ -154,7 +155,8 @@ export const withSecretForm = (SubForm, modal?: boolean) =>
     }
     onDataChanged(secretsData) {
       this.setState({
-        stringData: { ...secretsData.stringData },
+        stringData: { ...secretsData?.stringData },
+        base64Data: { ...secretsData?.base64Data },
```

Comment:
I would just have one `data` state property and always base64 encode the data. It keeps the logic simpler and works for plain strings, too.

Reply:
ok, sounds good, updated to have only one data state @spadgett